### DISCUSSION TITLE
Welcome to XCVario Discussions!

### DISCUSSION THREAD

**Discussion by iltis42** [iltis42/XCVario#175] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nferruzzi** [comment]
Hello! I'm an happy and satisfied xcvario 2022 owner actually mounted on a ls3/a ; on my dream box I'd like to build a mirror app for iOS/android to display the informations a bit better .. has anyone thought about this? Thanks in advance

**Reply by iltis42** [reply]
Thanks for sharing your idea here. 
What is exactly the idea for improvements to the display of informations ?

**Reply by nferruzzi** [reply]
More or less Id like to mirror the informations from the vario to the mobile displays .. they could even be close or resemble your official pages but by using more space(displays are bigger) the overall UI/UX feeling should improve. This is a screen from another app: the attitude indicator could be replaced with the vario screen (but this is just an example) or the g-force screen. Actually Id like to have it customisable in a way that all the informations you share could be ported over the companion app and why not with a map below :) . Of course a navigator would explode in complexity so for starter it could just display the vario, g-force and the inertial system maybe providing a simple access to the new firmwares to help updating the vario. 

![simulator_screenshot_396B1268-DD09-4DB8-9899-BB845C5B9E12](https://user-images.githubusercontent.com/1319874/162206343-16605a9d-6d9c-43e9-a6c8-9402a9832a8c.png)

**Comment by iltis42** [comment]
To develop your own mobile application there is nothing special. Today we can interwork with TopHat, XCSoar, LK8000, and whatever comes.
